working/augmentation-libraries/feature-specification.md

```diff
@@ -145,6 +145,17 @@ It is a compile-time error if:
 *   A library augmentation contains a normal `library` directive. They are not
     self-contained libraries, only pieces of the augmented library.
 
+*   An `import augment` directive has a `<uri>` that denotes an entity which
+    is not an augmentation library. *For example, it can not be a library.*
+    
+*   An `import` (not `import augment`) directive has a `<uri>` that denotes
+    an entity which is not a library. *For example, it cannot be an 
+    augmentation library.*
```

Comment:
Can we make this simpler and more explicit by just saying:

> An `import` refers to an entity which is a library augmentation or part file and not a library.

Also, we probably want to say "`export` or `import`".

Reply:
Done.